### DISCUSSION TITLE
rAudio 1 • 20210627: Only "Add" available in file mode

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#419] [Fixed Issues]
**rAudio 1 • 20210627**

Traversing the tree in the library window all functions are gone but "Add". No "Replace", no "Update database",  no "Update thumbnails".

**Comment by EarsofTin** [comment]
Same issue immediately following 20210627 update.

rAudio 1 • 20210627
5.10.27-3-ARCH aarch64
Raspberry Pi 3 Model B Plus Rev 1.3
BCM2837B0 • 953MiB
4 x Cortex-A53 @ 1.4GHz

**Reply by klassikmann** [reply]
Fixed in **rAudio 1 • 20210628**
Thank you.
